geomesa-compute/src/main/scala/org/locationtech/geomesa/compute/spark/GeoMesaSpark.scala

```diff
@@ -68,20 +87,43 @@ object GeoMesaSpark {
 
     ConfiguratorBase.setConnectorInfo(classOf[AccumuloInputFormat], conf, ds.connector.whoami(), ds.authToken)
 
-    if(useMock) ConfiguratorBase.setMockInstance(classOf[AccumuloInputFormat], conf, ds.connector.getInstance().getInstanceName)
-    else ConfiguratorBase.setZooKeeperInstance(classOf[AccumuloInputFormat], conf, ds.connector.getInstance().getInstanceName, ds.connector.getInstance().getZooKeepers)
-
+    if (useMock){
+      ConfiguratorBase.setMockInstance(classOf[AccumuloInputFormat],
+        conf,
+        ds.connector.getInstance().getInstanceName)
+    } else {
+      ConfiguratorBase.setZooKeeperInstance(classOf[AccumuloInputFormat],
+        conf,
+        ds.connector.getInstance().getInstanceName,
+        ds.connector.getInstance().getZooKeepers)
+    }
     InputConfigurator.setInputTableName(classOf[AccumuloInputFormat], conf, ds.getSpatioTemporalTable(sft))
     InputConfigurator.setRanges(classOf[AccumuloInputFormat], conf, qp.ranges)
-    qp.iterators.foreach { is => InputConfigurator.addIterator(classOf[AccumuloInputFormat], conf, is) }
+    qp.iterators.foreach { is => InputConfigurator.addIterator(classOf[AccumuloInputFormat], conf, is)}
```

Comment:
[picky] not worth changing, but just so you know in the future - we keep the extra space before the ending bracket.